src/components/message.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../styles/components/message.scss';
+import WarningIcon from '../svg/warning.svg';
+import CloseIcon from '../svg/times.svg';
+
+const MessageLevel = {
+  warning: 'warning',
+  failure: 'failure',
+  success: 'success',
+  info: 'info',
+};
+
+const iconSize = '1.125em';
+
+const Message = ({ children, level, onDismiss }) => (
+  <section className={`alert alert--${level}`}>
```

Comment:
```suggestion
  <section className={`alert alert--${level}`} role="status">
```
For ARIA reasons, [see here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_status_role)